clients/gtest/hipblaslt_gtest_ext_op.cpp

```diff
@@ -456,6 +488,56 @@ void AMaxTestWithScale(hipDataType    type,
     hipErr = hipFree(gpuInputScale);
 }
 
+template <typename Ti, typename To>
+void ValueDividedbyAMaxWithRcpTest(hipDataType type, hipDataType dtype, std::size_t m, std::size_t n, size_t value)
+{
+    std::size_t numElements = m * n;
+
+    To* gpuOutput{nullptr};
+    To* gpuOutputRcp{nullptr};
+    Ti* gpuInput{nullptr};
+    Ti* gpuWorkSpace{nullptr};
+    std::int32_t* gpuSync{nullptr};
+
+    auto hipErr = hipMalloc(&gpuOutput, sizeof(To));
+    hipErr      = hipMalloc(&gpuOutputRcp, sizeof(To));
+    hipErr      = hipMalloc(&gpuInput, m * n * sizeof(Ti));
+    hipErr      = hipMalloc(&gpuWorkSpace, 4096 * sizeof(Ti));
+    hipErr      = hipMalloc(&gpuSync, sizeof(std::int32_t));
+
+    std::vector<To> refOutput(1, -999.f);
+    std::vector<To> refOutputRcp(1, -999.f);
+    std::vector<To> cpuOutput(1, -999.f);
+    std::vector<To> cpuOutputRcp(1, -999.f);
+    std::vector<Ti> cpuInput(m * n, 0.f);
+
+    hipblaslt_init_hpl(cpuInput, m * n, 1, m * n);
+
+    hipErr = hipMemcpyHtoD(gpuInput, cpuInput.data(), m * n * sizeof(Ti));
+    hipErr = hipMemset(gpuWorkSpace, 0, m * n * sizeof(Ti));
+    hipErr = hipMemset(gpuSync, 0, sizeof(std::int32_t));
+
```

Comment:
LGTM but just a minor suggestion: Why don't we use CHECK_HIP_ERROR on these hipErr. I believe it would make it more robust.
![image](https://github.com/user-attachments/assets/d6a28d04-10eb-4ff3-bbce-b4e2fda4b071)

Reply:
good suggestion